uiowa/uiowa
# Feature: .htaccess patching

Resolves #2116 

Adds a process for patching the `.htaccess` file that is scaffolded to preserve our redirects. This may still result in the `.htaccess` file being changed and needing to be committed, but it should still have our changes.

# To test
* In `composer.json`, in the patches section for `drupal/core`, add an empty extra patch. It can literally be something like `"foo": "bar"`.
* Run `composer update --lock`, but interrupt it with `Ctrl-C` after it completes the `Removing drupal/core (8.9.3)` step.
* Remove your change to `composer.json` (or revert it).
* Run `composer update --lock` again.
* Observe that the patch applies after everything else has run.
* Do not commit any changes. The lock file will likely be given a new hash, but it is not necessary to add that change to this PR.
